.changeset/slimy-cups-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/celocli': patch
+---
+
+Add deprecation notice about future removal of account:offchain-read and account:offchain-write commands.These were created to showcase cip8 which has been abandond and they are presenting a high maintainence burden.
```

Comment:
Nit: 
- Adds code fences
- Adds link to [CIP8](https://github.com/celo-org/celo-proposals/blob/8260b49b2ec9a87ded6727fec7d9104586eb0752/CIPs/cip-0008.md)

```suggestion
Add deprecation notice about future removal of `account:offchain-read` and `account:offchain-write` commands. These were created to showcase ["CIP8: Expand Metadata to general off-chain storage"](https://github.com/celo-org/celo-proposals/blob/8260b49b2ec9a87ded6727fec7d9104586eb0752/CIPs/cip-0008.md), which has been abandond and they are presenting a high maintainence burden.
```